source/MaterialXGenShader/ShaderNode.cpp

```diff
@@ -32,6 +32,11 @@ string ShaderPort::getFullName() const
     return (_node->getName() + "_" + _name);
 }
 
+string ShaderPort::getValueString() const
```

Comment:
Sorry to keep hassling you on niggly things :), 
- For completeness, this can be exposed by adding an entry in the Python wrapper in `PyShader.cpp` in the `PyMaterialXGenShader` module.
- Also I think you can return a const reference to a string in this method.

Reply:
I'll add a python wrapper, but I don't think this can return a const reference, given that `Value::getValueString()` returns a `string` by value.